moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch

```diff
@@ -46,13 +47,16 @@
   <!-- Planning Functionality -->
   <include ns="move_group" file="$(find [GENERATED_PACKAGE_NAME])/launch/planning_pipeline.launch.xml">
     <arg name="pipeline" value="$(arg pipeline)" />
+    <param name="capabilities" value="$(arg capabilities)"/>
+    <param name="disable_capabilities" value="$(arg disable_capabilities)"/>
```

Comment:
I've just ran into a problem with `roslaunch` complaining that `param` is unexpected here in an `include`. And it's correct.

Should this be `arg` instead? Seems this is a problem on Melodic, which doesn't have #2127, which corrected this.

Reply:
#1893 introduced this in `master` ([here](https://github.com/ros-planning/moveit/commit/cf218879dbc23aadf88dd56b8abe7970b7d61030#diff-c9666b8ddf7fb2b57118595db82fca3bb31b167fb07876b124ca6a6f94ba33df)), but that got overwritten by #2127 ([here](https://github.com/ros-planning/moveit/commit/b0a4173c72a9376b1b3a8c63e9bfcc26626184eb#diff-c9666b8ddf7fb2b57118595db82fca3bb31b167fb07876b124ca6a6f94ba33df)).

All MoveIt configuration packages generated by the MSA on Melodic seem to be broken by this.

Reply:
An issue has better visibility: https://github.com/ros-planning/moveit/issues/2675.